pyproject.toml

```diff
@@ -4,6 +4,7 @@ package-mode = false
 [project]
 name = "auto-archiver-api"
 description = "API wrapper for Bellingcat's Auto Archiver, supports users, groups, sheet and url archives."
+version = "0.1.0"
```

Comment:
```suggestion
```
let's remove it from here and solve it in the other PR

Reply:
Without it, I was running into install errors. I could try again tho.

Reply:
Seems we're good without it. Removed!